tests/test_evented_decorator.py

```diff
@@ -245,3 +248,28 @@ class Foo:
 
     assert get_evented_namespace(Foo) == "my_events"
     assert is_evented(Foo)
+
+
+def test_name_conflicts() -> None:
+    # https://github.com/pyapp-kit/psygnal/pull/269
+
+    @evented
+    @dataclass
+    class Foo:
+        name: str
+        all: bool = False
+
```

Comment:
Add more conflicting fields

```python
        is_uniform: bool = True
        signals: list = field(default_factory=list)
       _psygnal_signals: str = "signals"
```

Reply:
Mmm, I was expecting `type(group).is_uniform` to be the classmethod but it a `Signal`.
Now that we have a safe way of getting the Signals with `__getitem__` maybe we can keep the few conflicting properties/methods from being overwritten by the Signals. For now there is only `all` and `is_uniform`.
What do you think?

Reply:
i'm going to rename `is_uniform` to `psygnals_uniform`, and deprecate `is_uniform`.  If a dataclass uses it as a name, it takes precedence in all regards.